packages/components/src/card/card-divider/component.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import { contextConnect } from '../../ui/context';
+import { View } from '../../view';
+import { useCardDivider } from './hook';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<{}, 'hr'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function CardDivider( props, forwardedRef ) {
+	const { as = 'hr', ...otherProps } = useCardDivider( props );
+
+	// Should it add the `components-card__divider` className ?
+	return <View { ...otherProps } as={ as } ref={ forwardedRef } />;
```

Comment:
Is this the correct way of using a `hr` HTML element for the Divider? (fallback value of the `as` prop to `hr`).

Also notice how the props have been typed as `PolymorphicComponentProps<{}, 'hr'>`